examples/example_plugin/example_plugin/views.py

```diff
@@ -77,6 +78,18 @@ class ExampleModelUIViewSet(views.NautobotUIViewSet):
     serializer_class = serializers.ExampleModelSerializer
     table_class = tables.ExampleModelTable
 
+    @action(detail=False, methods=["get"], url_path="all_names")
```

Comment:
For consistency the url_path should be (if possible) `all-names` since we normally use hyphens rather than underscores in our URLs.